### ISSUE TITLE
Add Mode : dry, fan for thermostat_mode

### ISSUE THREAD

**Issue by kentaclub** [athombv/node-homey-lib#127]
I'm writing a air conditioner control application, I use capabilities  "Thermostat_mode" by Homey but it doesn't have fan and dry mode.
If I write capabilities run separately then HomeKit will not pick up.

**Comment by TedTolboom** [comment]
Simply adding modes is not the solution @WeeJeWel; it will add additional modes for all thermostats that are not used by many devices... leading to a lot of confusions / issues with the end-user.

We need an option to be able to extend or overrule the current enum list for standard capabilities (with flow card options matching the actual capabilities).

In this way, we are do not need to create custom capabilities simply because the enum is not matching and can use the default capability list

**Comment by WeeJeWel** [comment]
Yes, I agree. I want a capability option that lets you specify the IDs that are available in the enum. We can then default this to the current set.

I think this might be added somewhere after Homey v2.0

**Comment by RobinBol** [comment]
Adding custom enum capability options is possible for `thermostat_mode` as of Homey v12.0.1: https://apps.developer.homey.app/the-basics/devices/capabilities#enum-capability-options
